app/services/kit_create_service.rb

```diff
@@ -1,8 +1,18 @@
 class KitCreateService
   include ServiceObjectErrorsMixin
 
+  KIT_BASE_ITEM_ATTRS = {
+    name: 'Kit',
+    category: 'kit',
+    partner_key: 'kit'
+  }
+
   attr_reader :kit
 
+  def self.FindOrCreateKitBaseItem!
```

Comment:
Idiomatic Ruby uses snake\_case: `find_or_create_kit_base_item!`.

Reply:
changed this in PR #4665 [a74dcc9](https://github.com/rubyforgood/human-essentials/pull/4665/commits/a74dcc9b2d89e90b410404aef99ee07c6af694e4)